doc/sections/module.md

```diff
@@ -1,45 +1,84 @@
 Module system
 -------------
 
-Lambdapi has a (very) light module system based on file paths. Although it has
-several limitations,  the module system allows the splitting developments into
-several files, and even several folders. The main thing to know is that a file
-holds a single module, which name corresponds to that of the file. For example
-a file `nat.lp` in the working directory will define the `nat` module,  and it
-can be referred as such in other files (or modules).
-
-However, note that `nat.lp` only defines module `nat` if `lambdapi` is invoked
-in the directory containing the file. Now, if the current directory contains a
-directory `lib`, itself containing a file `boolean.lp`, then the corresponding
-module is accessed using a full qualification `lib.boolean`. To illustrate the
-behaviour of the system, consider the following example.
-```bash
-working_directory  # lambdapi is invoked in that directory
-├── main.lp        # this module must always be refered as "main"
-├── bool.lp        # this module must always be refered as "bool"
-├── church
-│   ├── nat.lp     # this module must always be refered as "church.nat"
-│   └── list.lp    # this module must always be refered as "church.list"
-└── scott
-    ├── nat.lp     # this module must always be refered as "scott.nat"
-    └── list.lp    # this module must always be refered as "scott.list"
+Lambdapi has a (very) light module system based on file paths. It allows us to
+split our developments into several files, and even several folders.  The main
+thing to know is that a file holds a single module,  and accessing this module
+in other files requires using its *full* module path.
+
+#### Module path
+
+The module path that corresponds to a source file is defined using the name of
+the file, but also the position of the file under the *library root* (a folder
+under which the standard library and all packages are installed).
+
+The typical case is when we want to access a module of some installed library,
+for example the standard library. In that case, the module path is built using
+the file path as follows: if our source file is at `<LIB_ROOT>/std/bool.lp` it
+is accessed with module path `std.bool`.  And if there are nested folders then
+the module path gets more members. File `<LIB_ROOT>/std/a/b/c/d.lp` has module
+path `std.a.b.c.d`.
+
+By default, every modules are looked up under the library root. However, there
+are cases where the files we want to work with are not (yet!) placed under the
+library root.  The typical case is when a library (or package) is still  under 
+development. In that case,  what we can do is map the development folder under
+the library root, similarly to what would happen when mounting a volume in our
+file system. There are two ways of doing that, the first one being to used the
+`--map <MOD_PATH>:<DIR>` command line option.  However, the best way is to use
```

Comment:
`MOD_PATH` is the path as used in lambdapi while `DIR` is the filesystem path, is it?

Reply:
Yes, exactly.